server/game/cards/locations/04/renlyspavilion.js

```diff
@@ -0,0 +1,54 @@
+const DrawCard = require('../../../drawcard.js');
+
+class RenlysPavilion extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Kneel this card',
```

Comment:
This needs a different title as the default action for clicking on a card is to kneel it.  Something like 'Kneel this card to modify STR on 2 characters'